### ISSUE TITLE
pip install hangs, waiting for input on stdin

### ISSUE THREAD

**Issue by mdickinson** [mmaelicke/scikit-gstat#9]
When I run `pip install scikit-gstat` in a newly-created Python 3.6 environment, it appears to hang indefinitely. It looks as though it's waiting for user input in response to the license display; I need to type `q` and return twice to make the install complete.

Here's the transcript from the session (OS X 10.13, Python 3.6.0, pip 9.0.1), including the two `q` characters that I needed to type.
```
(testing-gstat)bash-3.2$ pip install scikit-gstat
Collecting scikit-gstat
  Using cached scikit-gstat-0.1.5.tar.gz
q
Requirement already satisfied: scipy in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from scikit-gstat)
Requirement already satisfied: numpy in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from scikit-gstat)
Requirement already satisfied: pandas in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from scikit-gstat)
Requirement already satisfied: nose in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from scikit-gstat)
Requirement already satisfied: python-dateutil>=2 in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from pandas->scikit-gstat)
Requirement already satisfied: pytz>=2011k in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from pandas->scikit-gstat)
Requirement already satisfied: six>=1.5 in /Users/mdickinson/.edm/envs/testing-gstat/lib/python3.6/site-packages (from python-dateutil>=2->pandas->scikit-gstat)
Installing collected packages: scikit-gstat
  Running setup.py install for scikit-gstat ... -q
done
Successfully installed scikit-gstat-0.1.5
```

**Comment by mmaelicke** [comment]
Thanks for reporting!   
Just realizing, happens on my Windows (10 Home) and Linux (Mint 17.2) as well and didn't happen a few weeks ago. I'll have a look on that one. I am not sure which module is triggering the license aggreement...

**Comment by mmaelicke** [comment]
Aaaargh!.
Finally found it. I didn't pass a license string to the `setup` function but called the native `license` Python  function for reasons I cannot understand anymore. 
Fixed it with last commit [#06f6ebd](https://github.com/mmaelicke/scikit-gstat/commit/06f6ebd). This fix is also uploaded to PyPI.
Thanks, again!

**Comment by mdickinson** [comment]
Thanks for the quick fixes (for this and for #10)!
